l/llvmlite/build_info.json

```diff
@@ -1,15 +1,15 @@
 {
-  "maintainer": "stutiibm",
-  "package_name": "llvmlite",
-  "github_url": "https://github.com/numba/llvmlite",
-  "version": "v0.41.1",
-  "default_branch": "main",
-  "package_dir": "l/llvmlite",
-  "build_script": "llvmlite_ubi_8_7.sh",
-  "docker_build": false,
-  "validate_build_script": true,
-  "use_non_root_user": false,
-  "v0.41.1, v0.41.0": {
-    "build_script": "llvmlite_ubi_8_7.sh"
+   "maintainer": "npanpaliya",
+   "package_name": "llvmlite",
+   "github_url": "https://github.com/numba/llvmlite",
+   "version": "v0.41.1",
```

Comment:
Can add support for latest version - v0.43.0

Reply:
For currency the latest version is added and if we need a specific version (that does not work with the same script) then we add another script with the required version.
Sample PR - https://github.com/ppc64le/build-scripts/pull/4493